app/build.gradle

```diff
@@ -2,6 +2,8 @@ import org.apache.tools.ant.taskdefs.condition.Os
 
 plugins {
     id 'com.android.application'
+    id 'kotlin-android'
+    id 'kotlin-android-extensions'
```

Comment:
Just wondering what this was used for?

Reply:
I'll delete this I was testing something for my MVP, but it didn't need it since it work without it